ebusd-2.1.x/en/vaillant/15.700.csv

```diff
@@ -51,6 +53,8 @@ r,,DisplayedOutsideTemp,outside temperature,,,,7300,,,tempv,,,Außentemperatur
 r;w,,SFMode,SF mode,,,,7400,,,sfmode,,,
 r;w,,KeyCodeforConfigMenu,code for installer menu,,,,7600,,,UIN,,,000-999; code for the installer's menues
 r;w,,OpMode,operation mode,,,,7b00,,,opmode,,,operation mode of all/configured zone(s) matching OpModeEffect
+r;w,,OpModeCooling,operation mode for cooling,,,,7c00,,,opmode,,,operation mode for cooling of all/configured zone(s)
+r;w,,OpModeVentilation,operation mode for ventilation,,,,7d00,,,opmode,,,operation mode for cooling of all/configured zone(s)
```

Comment:
please respond to @michlimek comments for this

Reply:
Confirmed the @michlimek version of OpModeVentilation